src/ui/RunEditor.tsx

```diff
@@ -1858,6 +1872,18 @@ export class RunEditor extends React.Component<Props, State> {
     }
 
     private async downloadSplits<T>(apiRun: Run<T>, apiUri: string) {
+        // TODO: Determine whether there are any unsaved changes in the Run Editor
```

Comment:
I think being able to close the Run Editor without disposing it would allow us to handle this?